app/views/shared/_nav_auth.html.slim

```diff
@@ -12,7 +12,10 @@ nav.bg-white
           = link_to t('shared.nav_auth.news'), root_path, class: 'mr3 gray'
           = link_to t('shared.nav_auth.contact'), root_path, class: 'mr3 gray'
       .sm-col-right.sm-right-align
-        div = t('shared.nav_auth.welcome', email: current_user.email)
+        div
+          <strong>
```

Comment:
This will bold the word welcome as well, I think we might need to modify the translation string to just be the word "Welcome" and not include the email variable.  That way we have two separate strings and can just bold the email address.